layouts/partials/openapi/render-object-table.html

```diff
@@ -67,6 +67,9 @@
 
       TODO: support `patternProperties` here.
     */}}
+    {{ if eq .additionalProperties true }}
+      {{ errorf "`additionalProperties: true` is de facto the default for Matrix and doesn't provide any information" }}
+    {{ end }}
```

Comment:
If we're going to do this, maybe update the comment above which claims "nothing to do"?

Reply:
Actually, I think we don't even need this anymore. My original premise was that we wanted to prevent `additionalProperties: true` because it's not getting rendered in the spec. However, as we discussed, `additionalProperties: true` doesn't add any information on its own. So not rendering it is not problematic. Therefore, I have removed this from the PR.